Cryptopia/Cryptopia.swift

```diff
@@ -232,6 +256,22 @@ public struct Cryptopia {
             }
         }
         
+        public func getOpenOrders(completion: @escaping (ResponseType) -> Void) {
+            let apiType = Cryptopia.API.getOpenOrders
+            
+            if apiType.checkInterval(response: store.openOrdersResponse) {
+                completion(.cached)
+            } else {
+                cryptopiaDataTaskFor(api: apiType) { (json, response, error) in
```

Comment:
When you pull the new changes, you would see the new ResponseType as the parameter of dataTask completion block here. Could you fix this?